kolibri/plugins/management/assets/src/vue/class-enroll-page/index.vue

```diff
@@ -353,5 +354,63 @@
 
   .col-name, .col-username
     width: 45%
+    
+  .results-text
+    font-size: 0.9375rem
+    
+  .toolbar
+    margin-top: 30px
+    
+  .toolbar:after
+    content: ''
+    display: table
+    clear: both
+    
+  input[type='search']
```

Comment:
Second option sounds like it could make things easier for you all later on? Would it be useful if I comment what files the searchbar styles are shared across in that .styl file too?